getCalspec/getCalspec.py

```diff
@@ -58,10 +58,18 @@ def get_calspec_keys(star_label):
     1      False
     ...
     """
-    label = star_label.upper()
+    label = star_label.upper().replace(' ','')
```

Comment:
General comment: you are calling `.upper()` and then `.replace(' ', '')` in a few places. I would generally write a helper function for something like this called something like `sanitizeString()` or something like that, because 1) it makes it easier to see, when reading the code, why you're doing it, and 2) means that if you want to go to a different santization in the future (like using `lower()` or going via a complicated function or whatever) you then only have to change it one place, so you get extra robustness for free. It's not _necessary_ to change it, but this is just a general "how to write cleaner code" comment really.